app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/JupiterSwapFragment.kt

```diff
@@ -17,24 +25,51 @@ import org.p2p.wallet.R
 import org.p2p.wallet.common.mvp.BaseMvpFragment
 import org.p2p.wallet.databinding.FragmentJupiterSwapBinding
 import org.p2p.wallet.swap.ui.jupiter.main.widget.SwapWidgetModel
+import org.p2p.wallet.utils.args
 import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+import java.util.*
```

Comment:
Something is broken with your code style. 
Are you sure you are using `Project`?